doc/guides/bluetooth/gatt-pics.rst

```diff
@@ -183,6 +186,9 @@ TSPC_GATT_4_22	True		Server: Write Long Characteristic
 TSPC_GATT_4_23	True		Server: Service Changed Characteristic (C.1)
 TSPC_GATT_4_24	False (*)	Server: Configured Broadcast (C.5)
 TSPC_GATT_4_25	False (*)	Server: Execute Write Request with empty queue (C.7)
+TSPC_GATT_4_26	False (*)	Server: Client Supported Features Characteristic
+				(C.9)
+TSPC_GATT_4_27	False (*)	Server: Database Hash Characteristic (C.8)
```

Comment:
This should be true, since we support it.

Reply:
FYI, TSPC_GATT_4_27 enables:
```
GATT/SR/GAS/BV-02-C
GATT/SR/GAS/BV-04-C
GATT/SR/GAS/BV-05-C
GATT/SR/GAS/BV-06-C
GATT/SR/GAS/BV-07-C
```